index.js

```diff
@@ -231,6 +232,8 @@ module.exports = {
     'prefer-template': 2,
     'require-yield': 0,
     'keyword-spacing': 2,
+    'react/display-name': 0,
```

Comment:
This seems fine to me

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/display-name.md

In libraries this could be necessary (such as snapkit) when composition does magic based on what children are present